persistence/src/main/java/com/epam/esm/BaseDao.java

```diff
@@ -0,0 +1,41 @@
+package com.epam.esm;
+
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * The interface Base dao.
```

Comment:
here is the name of the class but not it's description but javadoc is for description

Reply:
Fixed